config/crd/bases/rhdh.redhat.com_backstages.yaml

```diff
@@ -292,7 +292,7 @@ spec:
                       configuration
                     type: string
                   localDbConfig:
-                    description: Name of ConfigMap containing LocalDb (PostgreSQL)
+                    description: Name of ConfigMap containing LocalDb (P|ostgreSQL)
```

Comment:
It is "P|ostgreSQL", i.e. the same as it used to be, no?

Reply:
I agree it would be consistent with what we have in [`backstage_types.go`](https://github.com/janus-idp/operator/blob/1.1.x/api/v1alpha1/backstage_types.go#L183). But my point was to address this in a separate PR, as it is not related to https://issues.redhat.com/browse/RHIDP-1701

Reply:
outdated